reference_guide/ui_themes/themes_customize.md

```diff
@@ -166,6 +178,49 @@ The following example sets the background color for all labels to the color #F6E
 ```
 The `Label.background` entry supersedes, in the narrower context of label backgrounds, any default color as well as any wildcard color assigned to backgrounds.
 
+#### Customizing the Color of UI Tabs
+The color of UI Tabs are changed by adding a key-value pair to the `"ui": {}` section of a Theme description file.
+The `key` is the `element.property` format and the `value` is the custom color.
+
+There are two implementations of tabs in the IntelliJ Platform:
+* Editor Tabs, which e.g., represent open files in the [Editor window](https://www.jetbrains.com/help/idea/using-code-editor.html), and in [Tool Window bars](https://www.jetbrains.com/help/idea/tool-windows.html#bars_and_buttons).
+* Tabbed Panes, which e.g., are used for the [Run/Debug Configurations dialog](https://www.jetbrains.com/help/idea/run-debug-configurations-dialog.html).
+
+The control keys for UI Tabs were expanded from build 191 to 192 of the IntelliJ Platform.
+The build 191 control keys are compatible with build 192 and later versions of the IntelliJ Platform. 
+
+| Build 191 Element | Build 192 Element | Description of 192 Element |
```

Comment:
let's use 2019.1 and 2019.2 instead of "technical" numbers, since theme authors are more aware of official release numbering scheme

Reply:
Agreed.
build 19* --> release 2019.*
b19* --> r2019.*